src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -188,6 +190,9 @@ private Slave prepareDockerPlatforms() throws Exception {
         if (s != null) {
             j.jenkins.removeNode(s);
         }
+        if (BourneShellScript.FORCE_SHELL_WRAPPER) {
+            BourneShellScript.FORCE_SHELL_WRAPPER = false;
```

Comment:
nit: could remove the `if` and just always set `BourneShellScript.FORCE_SHELL_WRAPPER = false;`

Reply:
Done in ef70f18.